ethereum/contracts/interfaces/relayer/IWormholeRelayer.sol

```diff
@@ -105,6 +266,30 @@ interface IWormholeRelayerSend is IWormholeRelayerBase {
         uint256 gasLimit
     ) external payable;
 
+    /**
+     * @notice Performs a send, and uses the current refund to fund the 'send'
```

Comment:
I think we want to make the main difference between forward an refund clear.

E.g. something along the lines of:
The same as send, except that
1. it only checks whether there are sufficient funds to execute all forwards taking both leftover gas and passed value into account
2. it's only executed after receiveWormholeMessages has completed and hence does not return a sequence number